cities/brühl.json

```diff
@@ -0,0 +1,63 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                    6.89742,
+                    50.82805
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Brühler Wochenmarkt",
+                "location": "Balthasar-Neumann-Platz",
+                "opening_hours": "Tu, Th, Sa 07:00-13:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    6.90468,
+                    50.82831
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Frischemarkt Innenstadt",
+                "location": "Uhlstraße",
+                "opening_hours": "Mi 08:00-19:00"
```

Comment:
Looks like our validation script does not reliable - it detected an error but did not fail.

> BRÜHL: Frischemarkt Innenstadt
> Error: Mi <--- (Bitte benutze die englische Abkürzung "We" für "mi".)
> Validation done. 0 warning(s), 1 error(s) detected.

Can you please fix the weekday?